### ISSUE TITLE
Homebridge-HomeSeer4 Plugin Preventing Homebridge from starting

### ISSUE THREAD

**Issue by VirtualPanther** [jvmahon/Homebridge-HomeSeer4#158]
Greetings,
I am not a coder, so I did not realize initially where to look for my problem. I just noticed that Siri kept saying device is unresponsive. Took a few days to realize that all of them were HS4 devices. I went to Homebridge and saw this in the logs, repeating non-stop:

`[homebridge-homeseer4] This plugin is taking long time to load and preventing Homebridge from starting. See https://git.io/JtMGR for more info.`

I restarted Homebridge - no change. Rebooted my NAS (i.e. where Homebridge is running) - no change. Rolled back to v.1.0.2.0 -- nothing. Updated to latest again - nothing. Definitely used to work flawlessly.
![Homebridge 4 Error](https://user-images.githubusercontent.com/94378647/141800582-816b2486-9d20-43a6-8db3-4508ab49037a.jpg)
..

**Comment by jvmahon** [comment]
The only suggestion I can give is to recheck each of the install parameters (IP address, port, ASCII Port on, etc.). as set out here:  https://github.com/jvmahon/Homebridge-HomeSeer4/wiki

The place I'd check first:  Is HomeSeer on a dynamic IP address? If so, could it have changed? Make sure that HomeSeer's IP matches what you have in the homebridge Config.json

See section 2.2, here:  https://github.com/jvmahon/Homebridge-HomeSeer4/wiki

**Comment by jvmahon** [comment]
No further response, so I presume you fixed this.

**Comment by VirtualPanther** [comment]
Odd, I actually responded a couple of days ago, not sure where my post is...
My response was that, after checking all of the parameters, they were all correct. So nothing t fix, unfortunately. Then, just on a whim, I deleted every single device that was added to HomeKit from HomeSeer. Immediately after restarting Homebridge, the error went away. I m assuming that an issue developed in communicating with one of those devices. I do not know for sure. Logs were extremely unhelpful to me, as I am assuming to you. The error listed was too generic to narrow down the cause easily. I appreciate the help, truly, as I have nowhere else to go. Homeseer is stuck at its current stage, with many modern integration comforts lacking. I use it a lot for the background automation, but for manual and voice control, its is downright ancient...
